universal-login-provider/src/ULIFrameProvider.ts

```diff
@@ -114,7 +131,20 @@ export class ULIFrameProvider {
 
     element.innerHTML = `
       <img src="${this.config.logoUrl}" alt="U" >
+      <div></div>
```

Comment:
Why not use id e.g. `unilogin-button-notification`?

Reply:
I removed id. Because when i want operate on unilogin-button-notification only when button exists. So i get this element from ulButtonElement